package.json

```diff
@@ -78,7 +78,7 @@
     "typescript": "^4.2.4"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": "^16.20.0"
```

Comment:
This would prevent the package from being used on v18. Was this what you intended? We use v18 and v20 in most of our repositories, and likely will be upgrading the clients to v18 soon.

Reply:
Probably a good idea to avoid making significant changes in these dependency bump PRs if possible. If it was just a dependency bump, this PR would have no impact upon users of this package. But this change makes it a breaking change.